packages/nextjs/components/scaffold-stark/Faucet.tsx

```diff
@@ -41,7 +41,7 @@ export const Faucet = () => {
   useEffect(() => {
     const checkChain = async () => {
       try {
-        const providerInfo = await publicClient.getBlock();
+        const providerInfo = await publicClient.getBlockWithTxHashes();
```

Comment:
it seems to me that this will be a heavier request, which we are not supposed to do when calling `checkChain()` only, why do we need `txHashes` ?